html/changelogs/MoondancerPony-cardoxrate.yml

```diff
@@ -0,0 +1,3 @@
+author: MoondancerPony 
+delete-after: True
+  - tweak: "Increases the metabolism rate of Cardox to .6u every tick (two seconds)."
```

Comment:
Invalid format of the changelog.
You need:


```
author: MoondancerPony 
delete-after: True
changes:
  - tweak: "Increases the metabolism rate of Cardox to .6u every tick (two seconds)."
```

Reply:
Gahhh.